src/main/java/io/jenkins/plugins/opentelemetry/OpenTelemetryRootAction.java

```diff
@@ -22,25 +22,30 @@
     private OpenTelemetrySdkProvider openTelemetrySdkProvider;
 
     public Optional<ObservabilityBackend> getFirstMetricsCapableObservabilityBackend() {
-        final Optional<ObservabilityBackend> observabilityBackend = Optional.ofNullable(pluginConfiguration.getObservabilityBackends().stream().filter(backend -> backend.getMetricsVisualizationUrlTemplate() != null).findFirst().orElse(null));
+        final Optional<ObservabilityBackend> observabilityBackend = pluginConfiguration.getObservabilityBackends()
```

Comment:
this file is unrelated cleanup, can back out if desired, was just in here debugging why a link wasn't working after updating the icons